pkg/config/config_template.yaml

```diff
@@ -1356,16 +1366,19 @@ api_key:
 
 # runtime_security_config:
   ## @param enabled - boolean - optional - default: false
+  ## @env DD_RUNTIME_SECURITY_CONFIG_ENABLED - boolean - optional - default: false
   ## Set to true to enable Cloud Workload Security (CWS).
   #
   # enabled: false
 
   ## @param fim_enabled - boolean - optional - default: false
+  ## @env DD_RUNTIME_SECURITY_CONFIG_FIM_ENABLED - boolean - optional - default: false
```

Comment:
I don't think this setting is bound to an env var. See [here](https://github.com/datadog/datadog-agent/blob/3a3bdd40ee5ecc2e8c99afef1cec906af9104e30/pkg/config/config.go#L890), it is only marked as known.